### PR TITLE
Fixes: #3516 Update and rename arch-linux.json to pacman-package-manager.json

### LINKED ISSUE #3516: Arch Linux Cheat Sheet: just for pacman package manager?
Some feedback from a DDG user:

> these are not arch linux cheat sheet. instead this could be called as the cheat sheet for pacman package manager

Should this cheat sheet + triggers be narrowed specifically for the pacman package manager?

---

IA Page:  http://duck.co/ia/view/arch_linux_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @direwolf424

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Fixes #3516 

@moollaza  please edit the IA page and change the cheat sheet id there...

cc: @tagawa @edgesince84 

---

Instant Answer Page: http://duck.co/ia/view/arch_linux_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @direwolf424

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
thanks for the PRs @gautamkrishnar - we will review each one soon!

**Comment by tagawa** [comment]
Thank you for this @gautamkrishnar. I think we should merge the changes in #3526 first and then apply this PR.

File: share/goodie/cheat_sheets/json/pacman-package-manager.json [hunk 1]
```diff
@@ -1,13 +1,16 @@
 {
-    "id": "arch_linux_cheat_sheet",
-    "name": "Arch Linux",
-    "description": "Arch Linux Commands For Pacman,Yaourt and Some Basic Linux Commands",
+    "id": "pacman_package_manager",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/pacman-package-manager.json]: Please could you change this to `pacman_package_manager_cheat_sheet`?

File: share/goodie/cheat_sheets/json/pacman-package-manager.json [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+    "name": "Pacman Package Manager",
+    "description": "Linux Commands For Pacman,Yaourt and Some Basic Linux Commands",
     "metadata": {
 		"sourceName": "ArchWiki",
 		"sourceUrl": "https://wiki.archlinux.org/index.php/Pacman"
 	},
     "aliases": [
         "arch",
+        "arch linux",
+        "Manjaro",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/pacman-package-manager.json]: Please could you change this to `manjaro`? (uppercase is not allowed in alias names)

File: share/goodie/cheat_sheets/json/pacman-package-manager.json [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+        "Antergos",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/pacman-package-manager.json]: Please could you change this to `antergos`?

File: share/goodie/cheat_sheets/json/pacman-package-manager.json [hunk 4]
```diff
@@ -1,13 +1,16 @@
 {
-    "id": "arch_linux_cheat_sheet",
-    "name": "Arch Linux",
-    "description": "Arch Linux Commands For Pacman,Yaourt and Some Basic Linux Commands",
+    "id": "pacman_package_manager",
+    "name": "Pacman Package Manager",
+    "description": "Linux Commands For Pacman,Yaourt and Some Basic Linux Commands",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/pacman-package-manager.json]: Please could you change this to `Linux commands For Pacman and Yaourt`?

**Comment by tagawa** [comment]
@gautamkrishnar I think we should remove the whole `Linux Command` section, because that should be covered in the [Linux cheatsheet](https://duckduckgo.com/?q=linux+cheat+sheet).

**Comment by gautamkrishnar** [comment]
@tagawa updated...

**Comment by gautamkrishnar** [comment]
I think this issue is already fixed?

**Pull request closed without merging**